app/views/career_site/positions/show.html.slim

```diff
@@ -9,7 +9,7 @@
 .row#position_heading.mt-3.gap-3.justify-content-between
   .col-12.col-md-10
     .col
-      h2 = @position.name
+      h1 = @position.name
```

Comment:
as i understand we should not make this change
"Heading: increase font-size from 20px to 24px (shouldn’t be applied for Application form)"

and as I can see for the index page it is already 24px

Reply:
Issue talks only about show.
[MockUp](https://www.figma.com/design/kQOH4s5DnfWL0p6o6tlMQz/Draft-%5BATS%5D?node-id=1520-10620&node-type=canvas&t=JhKHthbxjOk1CNyI-0)

Reply:
ok, agree. seems "shouldn’t be applied for Application form" is for `Apply for this position` text